data.js

```diff
@@ -0,0 +1,46 @@
+var sides = [
+"Miso Glazed Carrots",
+"Coleslaw",
+"Garden Salad",
+"Crispy Potatoes",
+"Sweet Potato Tots",
+"Coconut Rice",
+"Caeser Salad",
+"Shrimp Summer Rolls",
+"Garlic Butter Mushrooms",
+"Hush Puppies"];
+
+var mainDishes = [
+"Spaghetti and Meatballs",
+"Pineapple Chicken",
+"Shakshuka",
+"Thai Yellow Curry",
+"Bibimbap",
+"Chicken Parmesean",
+"Butternut Squash Soup",
+"BBQ Chicken Burgers",
+"Ramen",
+"Empanadas",
+"Chicken Fried Rice",
+"Sheet Pan Fajitas",
+"Margarita Pizza"];
+
+ var desserts = [
+"Apple Pie",
+"Lemon Meringue Pie",
+"Black Forest Cake",
+"Banana Bread",
+"Peach Cobbler",
+"Cheesecake",
+"Funfetti Cake",
+"Baklava",
+"Flan",
+"Macarons",
+"Macaroons",
+"Chocolate Cupcakes",
+"Pavlova",
+"Pumpkin Pie",
+"Key Lime Pie",
+"Tart Tatin",
+"Croissants",
+"Eclairs"]
```

Comment:
normal formatting for the end of an array like this is
```
"Eclairs"
];
```

Reply:
thank you, will edit this.